pwiz_tools/Skyline/Test/FindNodeTest.cs

```diff
@@ -49,7 +50,7 @@ public void RunTestFindNode(RefinementSettings.ConvertToSmallMoleculesMode asSma
 
             SrmDocument doc = CreateStudy7Doc();
             doc = new RefinementSettings().ConvertToSmallMolecules(doc, TestDirectoryName, asSmallMolecules);
-            var displaySettings = new DisplaySettings(null, false, 0, 0); //, ProteinDisplayMode.ByName);
+            var displaySettings = new DisplaySettings(new NormalizedValueCalculator(doc), null, false, 0, NormalizeOption.RatioToFirstStandard(doc.Settings)); //, ProteinDisplayMode.ByName);

```

Comment:
Seems overly complicated new code compared with supplying nodePep and false, 0, 0